### PR TITLE
patch: issue 566

### PR DESCRIPTION
**Pull request by jayu**
A few grammar mistakes corrected

### CODE REVIEW

File: docs/CONFIGURATION.md [hunk 1]
```diff
@@ -49,18 +49,21 @@ module.exports = {
...
-  
-  If `test` is omitted the rule is applicable for all files.
-  
-  The last matched rule is used for transformation. If the last matched action for a file is `"ignore"` the file will be evaluated as is, so that file must not contain any js code that cannot be executed in nodejs environment (it's usually true for any lib in `node_modules`).  
 
-  Information about `Evaluator`, its default and custom implementations can be founded it [evaluators section of How it works docs](../HOW_IT_WORKS.md#evaluators)
+  If `test` is omitted, the rule is applicable for all the files.
+
+  The last matched rule is used for transformation. If the last matched action for a file is `"ignore"` the file will be evaluated as is, so that file must not contain any js code that cannot be executed in nodejs environment (it's usually true for any lib in `node_modules`).
+  If you need to compile certain modules under `/node_modules/` (which can be the case in monorepo projects), it's recommended to do it on a module by module basis for faster transforms, e.g. `ignore: /node_modules[\/\\](?!some-module|other-module)/`. Example is using Regular Expressions negative lookahead.
```
Comment (jayu) [inline docs/CONFIGURATION.md:61]: I restored the deleted sentence about including particular module

**Comment by Anber** [comment]
Articles… I hate articles :)

Thank you!

**Comment by jayu** [comment]
A reading sometime after the writing helps identify most of the issues :D I hate articles too :P

**Comment by Anber** [comment]
I think it's because we don't have them in our native languages :)

**Pull request merged**